### PR TITLE
Add (stub) localization guide for Spanish

### PR DESCRIPTION
**Pull request by sftim**
[Rendered](https://deploy-preview-24477--kubernetes-io-master-staging.netlify.app/es/docs/contribute/#siguientes-pasos).

Copying some tips from https://github.com/kubernetes/website/issues/13948#issue-435641123

### CODE REVIEW

**Comment by raelga** [comment]
Thanks @sftim, the table was just a draft, far from "live" ready.

I will review it with the team and get a proper table. :)

**Comment by sftim** [comment]
How's this looking?

**Comment by raelga** [comment]
> How's this looking?

Sorry, I'll keep you posted.

**Comment by kbhawkey** [comment]
Is this PR ready for review?
/cc @kubernetes/sig-docs-es-reviews

File: content/es/docs/contribute/_index.md [hunk 1]
```diff
@@ -83,3 +83,7 @@ para proporcionar un punto de partida.
   - Proponer mejoras al sitio web de Kubernetes y otras herramientas
 
 
+## {{% heading "whatsnext" %}}
+
+También puede leer la
```
Comment (electrocucaracha) [inline content/es/docs/contribute/_index.md:88]:
```suggestion
También puedes leer la
```

File: content/es/docs/contribute/localization_es.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+---
+title: Contribuir a la documentación de Kubernetes en español
+content_type: concept
+---
+
+<!-- overview -->
+
+¡Bienvenida/o! Esta página es un esbozo.
```
Comment (electrocucaracha) [inline content/es/docs/contribute/localization_es.md:8]:
```suggestion
¡Bienvenido(a)! Esta página es un esbozo.
```

File: content/es/docs/contribute/localization_es.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+
+<!-- body -->
+
+## Glosario de terminología {#terminologia}
+
+|      English      |        Español         |  Género   |    Commentarios          |
+| ----------------- | ---------------------- | --------- | ------------------------ |
+| availability zone | zona de disponibilidad |           |                          |
+| bearer token      | ⚠ ?                    |           |                          |
```
Comment (electrocucaracha) [inline content/es/docs/contribute/localization_es.md:18]:
```suggestion
| bearer token      | bearer token                    |           |                          |
```

File: content/es/docs/contribute/localization_es.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+| built-in          | incorporados           |           |                          |
+| conditions        | estado                 |           | para node conditions     |
```
Comment (electrocucaracha) [inline content/es/docs/contribute/localization_es.md:20]:
```suggestion
| conditions        | condiciones                 |           | para node conditions     |
```

File: content/es/docs/contribute/localization_es.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+| container         | contendador            |           |                          |
```
Comment (electrocucaracha) [inline content/es/docs/contribute/localization_es.md:21]:
```suggestion
| container         | contenedor            |           |                          |
```

File: content/es/docs/contribute/localization_es.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+| controller        | controlador            |           |                          |
+| deploy            | desplegar              |           |                          |
+| Deployment        | Deployment             |           | objeto Kubernetes        |
+| Endpoints         | Endpoints              |           | objeto Kubernetes        |
+| file              | fichero                |           |                          |
```
Comment (electrocucaracha) [inline content/es/docs/contribute/localization_es.md:26]: This word is translated as `fichero` mostly in Spain but in Latin america is used `archivo`

File: content/es/docs/contribute/localization_es.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+| frontend          | frontend               |           |                          |
+| healthy           | operativo              |           |                          |
+| high availability | alta disponibilidad    |           |                          |
+| hook              | hook                   |           |                          |
+| instance          | instancia              |           |                          |
+| Lease             | Lease                  |           | objeto Kubernetes        |
+| Pod               | Pod                    | masculino | objeto Kubernetes        |
+| ratio             | ritmo                  |           |                          |
```
Comment (electrocucaracha) [inline content/es/docs/contribute/localization_es.md:34]:
```suggestion
| ratio             | radio                  |           |                          |
```

File: content/es/docs/contribute/localization_es.md (hunk 6 above)
Comment (raelga) [inline content/es/docs/contribute/localization_es.md:26]:
```suggestion
| file              | fichero/archivo                |           |                          |
```

File: content/es/docs/contribute/localization_es.md (hunk 7 above)
Comment (raelga) [reply content/es/docs/contribute/localization_es.md:34]: Sobre esta no estoy muy seguro, `ratio` se refiere a una `proporción` en este caso.

File: content/es/docs/contribute/localization_es.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+¡Bienvenido(a)! Esta página es un esbozo.
+
+
+<!-- body -->
+
+## Glosario de terminología {#terminologia}
+
+|      English      |        Español         |  Género   |    Commentarios          |
+| ----------------- | ---------------------- | --------- | ------------------------ |
+| availability zone | zona de disponibilidad |           |                          |
```
Comment (electrocucaracha) [inline content/es/docs/contribute/localization_es.md:17]:
```suggestion
| availability zone | zona de disponibilidad |  femenino         |                          |
```

File: content/es/docs/contribute/localization_es.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+| bearer token      | bearer token                    |           |                          |
```
Comment (electrocucaracha) [inline content/es/docs/contribute/localization_es.md:18]:
```suggestion
| bearer token      | bearer token                    |  masculino         |                          |
```

File: content/es/docs/contribute/localization_es.md [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+| built-in          | incorporados           |           |                          |
```
Comment (electrocucaracha) [inline content/es/docs/contribute/localization_es.md:19]:
```suggestion
| built-in          | incorporados           |  masculino         |                          |
```

File: content/es/docs/contribute/localization_es.md [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+| conditions        | condiciones                 |           | para node conditions     |
```
Comment (electrocucaracha) [inline content/es/docs/contribute/localization_es.md:20]:
```suggestion
| conditions        | condiciones                 |  masculino         | para node conditions     |
```

File: content/es/docs/contribute/localization_es.md [hunk 12]
```diff
@@ -0,0 +1,42 @@
...
+| container         | contenedor            |           |                          |
```
Comment (electrocucaracha) [inline content/es/docs/contribute/localization_es.md:21]:
```suggestion
| container         | contenedor            | masculino          |                          |
```

File: content/es/docs/contribute/localization_es.md [hunk 13]
```diff
@@ -0,0 +1,42 @@
...
+| controller        | controlador            |           |                          |
```
Comment (electrocucaracha) [inline content/es/docs/contribute/localization_es.md:22]:
```suggestion
| controller        | controlador            | masculino          |                          |
```

File: content/es/docs/contribute/localization_es.md [hunk 14]
```diff
@@ -0,0 +1,42 @@
...
+| deploy            | desplegar              |           |                          |
+| Deployment        | Deployment             |           | objeto Kubernetes        |
```
Comment (electrocucaracha) [inline content/es/docs/contribute/localization_es.md:24]:
```suggestion
| Deployment        | Deployment             |  masculino         | objeto Kubernetes        |
```

File: content/es/docs/contribute/localization_es.md [hunk 15]
```diff
@@ -0,0 +1,42 @@
...
+| Endpoints         | Endpoints              |           | objeto Kubernetes        |
```
Comment (electrocucaracha) [inline content/es/docs/contribute/localization_es.md:25]:
```suggestion
| Endpoints         | Endpoints              | masculino          | objeto Kubernetes        |
```

File: content/es/docs/contribute/localization_es.md [hunk 16]
```diff
@@ -0,0 +1,42 @@
...
+| file              | fichero/archivo                |           |                          |
```
Comment (electrocucaracha) [inline content/es/docs/contribute/localization_es.md:26]:
```suggestion
| file              | fichero/archivo                | masculno          |                          |
```

File: content/es/docs/contribute/localization_es.md [hunk 17]
```diff
@@ -0,0 +1,42 @@
...
+| frontend          | frontend               |           |                          |
```
Comment (electrocucaracha) [inline content/es/docs/contribute/localization_es.md:27]:
```suggestion
| frontend          | frontend               | masculino          |                          |
```

File: content/es/docs/contribute/localization_es.md [hunk 18]
```diff
@@ -0,0 +1,42 @@
...
+| healthy           | operativo              |           |                          |
+| high availability | alta disponibilidad    |           |                          |
+| hook              | hook                   |           |                          |
```
Comment (electrocucaracha) [inline content/es/docs/contribute/localization_es.md:30]:
```suggestion
| hook              | hook                   |  masculino         |                          |
```

File: content/es/docs/contribute/localization_es.md [hunk 19]
```diff
@@ -0,0 +1,42 @@
...
+| instance          | instancia              |           |                          |
```
Comment (electrocucaracha) [inline content/es/docs/contribute/localization_es.md:31]:
```suggestion
| instance          | instancia              | femenino          |                          |
```

File: content/es/docs/contribute/localization_es.md [hunk 20]
```diff
@@ -0,0 +1,42 @@
...
+| Lease             | Lease                  |           | objeto Kubernetes        |
```
Comment (electrocucaracha) [inline content/es/docs/contribute/localization_es.md:32]:
```suggestion
| Lease             | Lease                  |  masculino         | objeto Kubernetes        |
```

File: content/es/docs/contribute/localization_es.md (hunk 7 above)
Comment (electrocucaracha) [reply content/es/docs/contribute/localization_es.md:34]: y su genero seria `femenino`

File: content/es/docs/contribute/localization_es.md [hunk 21]
```diff
@@ -0,0 +1,42 @@
...
+| Pod               | Pod                    | masculino | objeto Kubernetes        |
+| ratio             | ritmo                  |           |                          |
+| runtime           | tiempo de ejecución    |           |                          |
```
Comment (electrocucaracha) [inline content/es/docs/contribute/localization_es.md:35]:
```suggestion
| runtime           | motor de ejecución    | masculino          | Container Runtime                         |
```

File: content/es/docs/contribute/localization_es.md [hunk 22]
```diff
@@ -0,0 +1,42 @@
...
+| scheduler         | planificador           |           |                          |
```
Comment (electrocucaracha) [inline content/es/docs/contribute/localization_es.md:36]:
```suggestion
| scheduler         | planificador           | masculino          |                          |
```

File: content/es/docs/contribute/localization_es.md [hunk 23]
```diff
@@ -0,0 +1,42 @@
...
+| Secret            | Secret                 | masculino | objeto Kubernetes        |
+| secret            | secreto                |           | información confidencial |
```
Comment (electrocucaracha) [inline content/es/docs/contribute/localization_es.md:38]:
```suggestion
| secret            | secreto                | masculino           | información confidencial |
```

File: content/es/docs/contribute/localization_es.md [hunk 24]
```diff
@@ -0,0 +1,42 @@
...
+| shell             | terminal               | masculino |                          |
```
Comment (electrocucaracha) [inline content/es/docs/contribute/localization_es.md:39]:
```suggestion
| shell             | terminal               | femenino |                          |
```

File: content/es/docs/contribute/localization_es.md [hunk 25]
```diff
@@ -0,0 +1,42 @@
...
+| stateless         | stateless              |           |                          |
+| taint             | contaminación          |           |                          |
+| worker node       | nodo de trabajo        |           |                          |
```
Comment (electrocucaracha) [inline content/es/docs/contribute/localization_es.md:42]:
```suggestion
| worker node       | nodo de trabajo        | masculino          |                          |
```

**Comment by kbhawkey** [comment]
:wave: @sftim .
What do you think about the reviewer's suggestions?

File: content/es/docs/contribute/localization_es.md (hunk 7 above)
Comment (sftim) [reply content/es/docs/contribute/localization_es.md:34]: I'm happy to include one of these suggestions - which should it be?

**Comment by sftim** [comment]
https://github.com/kubernetes/website/compare/393520080aec4c22033fcdee3e94f3bb1c64b14b..cf69eb89569d69401c686ff8a5ab5f9e6eec5b4a accepted most of the suggestions, leaving the “ratio” discussion unresolved.

This can go in as-is and get tidied up once merged, or I can take a suggestion about what the table row for “ratio” should look like. I'm definitely not qualified to have a strong opinion.

File: content/es/docs/contribute/localization_es.md (hunk 6 above)
Comment (raelga) [reply content/es/docs/contribute/localization_es.md]: Solved in https://github.com/kubernetes/website/pull/24477/commits/c1aa3cc7a1a249bf0a7f94a99d6963cf4766a65e

File: content/es/docs/contribute/localization_es.md (hunk 7 above)
Comment (raelga) [reply content/es/docs/contribute/localization_es.md]:
```suggestion
| ratio             | proporción             | femenino  |                          |
```

**Comment by raelga** [comment]
Thanks all for participating!

I think we can start with the current state of the document and evolve it over time.

/label tide/merge-method-squash
/lgtm

**Pull request merged**